core-bundle/contao/widgets/FileTree.php

```diff
@@ -381,11 +381,11 @@ protected function getPickerUrlExtras($values = array())
 	 *
 	 * @param File   $objFile
 	 * @param string $strInfo
-	 * @param string $blnRemovable
+	 * @param string $strClass
 	 *
 	 * @return string
 	 */
-	protected function getPreviewImage(File $objFile, $strInfo, $blnRemovable = false)
+	protected function getPreviewImage(File $objFile, $strInfo, $strClass='preview-image gimage')
```

Comment:
```suggestion
	protected function getPreviewImage(File $objFile, $strInfo, $strClass='gimage')
```

This is how it is in Contao 5.3. Are there good reasons to change a legacy class in a minor release?

Reply:
Other than forgetting to change it in the next major, there isn't a reason, no.

Reply:
Then let's keep using the `gimage` class for the time being.

Reply:
See 96c4a40bca8c6f76958158f1b4e1230badccba46